doc/source/reference/simd/simd-optimizations-tables-diff.inc

```diff
@@ -0,0 +1,51 @@
+.. generated via doc/source/reference/simd/simd-optimizations.py
+
+``x86::Intel Compiler unix-like`` - CPU feature names
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. table::
+    :align:left
+    =========== ======================================================================================================================
+    Name        Implies                                                                                                               
+    =========== ======================================================================================================================
+    ``FMA3``    ``SSE`` ``SSE2`` ``SSE3`` ``SSSE3`` ``SSE41`` ``POPCNT`` ``SSE42`` ``AVX`` ``F16C`` **``AVX2``**                      
+    ``AVX2``    ``SSE`` ``SSE2`` ``SSE3`` ``SSSE3`` ``SSE41`` ``POPCNT`` ``SSE42`` ``AVX`` ``F16C`` **``FMA3``**                      
+    ``AVX512F`` ``SSE`` ``SSE2`` ``SSE3`` ``SSSE3`` ``SSE41`` ``POPCNT`` ``SSE42`` ``AVX`` ``F16C`` ``FMA3`` ``AVX2`` **``AVX512CD``**
+    =========== ======================================================================================================================
+
+.. note::
+  The following features aren't supported by ``x86::Intel Compiler unix-like``:
+  **``XOP`` ``FMA4``**
+
+``x86::Intel Compiler msvc-like`` - CPU feature names
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. table::
+    :align:left
+    =========== ======================================================================================================================
+    Name        Implies                                                                                                               
+    =========== ======================================================================================================================
+    ``FMA3``    ``SSE`` ``SSE2`` ``SSE3`` ``SSSE3`` ``SSE41`` ``POPCNT`` ``SSE42`` ``AVX`` ``F16C`` **``AVX2``**                      
+    ``AVX2``    ``SSE`` ``SSE2`` ``SSE3`` ``SSSE3`` ``SSE41`` ``POPCNT`` ``SSE42`` ``AVX`` ``F16C`` **``FMA3``**                      
+    ``AVX512F`` ``SSE`` ``SSE2`` ``SSE3`` ``SSSE3`` ``SSE41`` ``POPCNT`` ``SSE42`` ``AVX`` ``F16C`` ``FMA3`` ``AVX2`` **``AVX512CD``**
+    =========== ======================================================================================================================
```

Comment:
Since the two intel compiler tables are the same, they can be combined.

Reply:
done